File: arrays_hasing/0058_Length_of_Last_Word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int res = 0;
        int i = s.length() - 1;
        while(s.charAt(i) == ' ') {
            i--;
        }
        while(i >= 0 && s.charAt(i) != ' ') {
            i--;
            res++;
        }

        return res;
    }
}

File: leetcode_150/0058_Length_of_Last_Word.go
func lengthOfLastWord(s string) int {
	i := len(s) - 1
	for i >= 0 && s[i] == ' ' {
		i--
	}

	result := 0
	for i >= 0 && s[i] != ' ' {
		result++
		i--
	}
	return result
}